README.md

```diff
@@ -150,9 +147,15 @@ await tick();
 expect(wrapper.find('CommitRow')).toHaveLength(2);
 ```
 
-Selectors:
+### Selectors
+
 If you are writing jest tests, you can use a Component (and Styled Component) names as a selector. Additionally, if you need to use a DOM query selector, use `data-test-id` instead of a class name. We currently don’t, but it is something we can use babel to strip out during the build process.
 
+### Undefined `theme` properties in tests
+
+Instead of using `mount()` from `enzyme...use this: `import {mountWithTheme} from 'sentry-test/enzyme'` so that the component under test gets wrapped with a [`<ThemeProvider>`](https://emotion.sh/docs/theming).
```

Comment:
@billyvg looks like this sentence is missing a backtick somewhere.

Reply:
Oops, fixed